usual/cfparser.c

```diff
@@ -127,9 +127,22 @@ static bool parse_ini_file_internal(const char *fn, cf_handler_f user_handler, v
 			break;
 
 		/* read key val */
-		key = p;
-		while (*p && (isalnum(*p) || strchr("_.-*", *p))) p++;
-		klen = p - key;
+		if (*p && *p == '\"'){
+			key = ++p;
+			while (*p && *p != '\"') p++;
```

Comment:
Should we add isalnum(*p) check here not to pass eol?

e.g.

while (*p && (isalnum(*p) || strchr("_.-*", *p)) && *p != '\"' ) p++;